sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobOutputStream.java

```diff
@@ -35,8 +36,24 @@
  */
 public abstract class BlobOutputStream extends StorageOutputStream {
 
+    private AtomicBoolean isAlreadyClosed;
+    private final boolean isMultipleClosedAllowed;
```

Comment:
I'm not sure we have to do this .
The Closeable contract says `Closes this stream and releases any system resources associated with it. If the stream is already closed then invoking this method has no effect.`

See https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html

Reply:
Agree. We don't need to have this fallback behavior. We can remove `isMultipleCloseAllowed` option.